_includes/api/en/4x/express.static.md

```diff
@@ -30,15 +30,19 @@ See also the [example below](#example.of.express.static).
 For more information, see [Serving static files in Express](/starter/static-files.html).
 and [Using middleware - Built-in middleware](/{{page.lang}}/guide/using-middleware.html#middleware.built-in).
 
-<h5 id='dotfiles'> dotfiles</h5>
+<h5 id='dotfiles'>dotfiles</h5>
 
 Possible values for this option are:
 
 - "allow" - No special treatment for dotfiles.
 - "deny" - Deny a request for a dotfile, respond with `403`, then call `next()`.
 - "ignore" - Act as if the dotfile does not exist, respond with `404`, then call `next()`.
 
-**NOTE**: With the default value, it will not ignore files in a directory that begins with a dot.
+{% capture dotfiles %}
+When this property is not set and so uses the default value, the behavior is similar to 'ignore', with the exception that files in a directory that begins with a dot are **NOT** ignored. This behavior is based on the 1.x version of [serve-static](https://expressjs.com/en/resources/middleware/serve-static.html) middleware.
```

Comment:
Actually it does ignore the dot files even on 1.x version.

Reply:
Does it ignore the directory in 1.x? E.g. `.foo/test.js`?